src/RobotsTxt.php

```diff
@@ -104,17 +104,17 @@ protected function isCommentLine(string $line): bool
 
     protected function isUserAgentLine(string $line): bool
     {
-        return strpos(trim($line), 'User-agent') === 0;
+        return strpos(trim($line), 'user-agent') === 0;
```

Comment:
Shouldn't we use `strtolower` on `$line` to make sure `$line` itself is lowercased?

Reply:
Yes, definitely.

And another one here, correct?
Line `115` - `src/RobotsTxt.php`
```php
 protected function parseDisallow(string $line): string
    {
        return trim(str_replace('disallow', '', trim($line)), ': ');
    }
```
to
```php
 protected function parseDisallow(string $line): string
    {
        return trim(str_replace('disallow', '', trim(strtolower($line))), ': ');
    }
```
If nothing else should be changed, I will do push the changes.